frontend/src/components/survey-handler/survey-handler.test.tsx

```diff
@@ -100,6 +100,38 @@ describe('SurveyHandler', () => {
       expect(nextBtn).toBeEnabled();
     });
 
+    it('should render the single choice label', () => {
+      const singleChoiceQuestion = questions[1];
+      const mockSurvey: Survey = {
+        ...surveyDefault,
+        questions: [singleChoiceQuestion],
+        current_answers_survey: {
+          ...surveyDefault.current_answers_survey,
+          questions: [singleChoiceQuestion],
+        },
+      };
+
+      const { getByText } = render(<SurveyHandler survey={mockSurvey} />);
+
+      expect(getByText(getSurveyTitle('Single Choice'))).toBeInTheDocument();
+    });
+
+    it('should render the multiple choice label', () => {
+      const singleChoiceQuestion = questions[2];
+      const mockSurvey: Survey = {
+        ...surveyDefault,
+        questions: [singleChoiceQuestion],
+        current_answers_survey: {
+          ...surveyDefault.current_answers_survey,
+          questions: [singleChoiceQuestion],
+        },
+      };
+
+      const { getByText } = render(<SurveyHandler survey={mockSurvey} />);
+
+      expect(getByText(getSurveyTitle('Multiple Choice'))).toBeInTheDocument();
```

Comment:
The best way to test this is rendering the component and check screen, with this you test all comportament, is unnecessary export the function.